easybuild/tools/filetools.py

```diff
@@ -384,12 +384,14 @@ def extract_file(fn, dest, cmd=None, extra_options=None, overwrite=False, forced
     return find_base_dir()
 
 
-def which(cmd, retain_all=False, check_perms=True):
+def which(cmd, retain_all=False, check_perms=True, log_ok=True, log_error=True):
     """
     Return (first) path in $PATH for specified command, or None if command is not found
 
     :param retain_all: returns *all* locations to the specified command in $PATH, not just the first one
     :param check_perms: check whether candidate path has read/exec permissions before accepting it as a match
+    :param log_ok: Log an info message where the command has been found (if any)
+    :param log_error: Log a warning message when command hasn't been found
```

Comment:
The intension was that 1) is the "OK path" and 2) the "Error path".

So IMO `log_error` is ok because it logs (a warning in case of) an error. Otherwise I'd need to rename `log_ok` to `log_info`?

Reply:
OK, thanks for clarifying.